engine/src/main/java/org/hibernate/validator/internal/metadata/raw/ConstrainedElement.java

```diff
@@ -47,7 +47,7 @@
 	 * @author Gunnar Morling
 	 */
 	enum ConstrainedElementKind {
-		TYPE, FIELD, CONSTRUCTOR, METHOD, PARAMETER
+		TYPE, CONSTRUCTOR, METHOD, PARAMETER, PROPERTY
```

Comment:
So that means `PROPERTY` is used for getters and fields, I suppose?

We would have to check this wrt the other changes regarding getter + field.

Reply:
Hey, this ties in a bit with my recent comment: the "raw" model was not meant to do abstractions from actual elements (fields, getters etc.). Adding `PROPERTY` to `ConstraintElementKind` blurries that line. I didn't look into the entire PR, so perhaps it makes sense, but I wanted to bring up that it appears as quite some deviation from the existing abstractions.

Reply:
+1 that's the same problem. Will work on it :)

Reply:
@gunnarmorling yup, we are well aware of the issue. That's the big thing that still needs fixing in this PR.